<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抽检助手</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/icon/image/icon.ico</normaloff>:/icon/image/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>991</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>获取列表</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="pageIndexEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="site_table"/>
     </item>
     <item>
      <widget class="QPushButton" name="getSiteListBtn">
       <property name="text">
        <string>获取清单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSiteTableBtn">
       <property name="text">
        <string>清空表格</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_login">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>登录获取Cookie</string>
    </property>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>易运维主界面登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cookieEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>JSESSIONID=6215F2A44F8F98C61194A5CDB1AA89EB; username=CHENRONGHUI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getCookieBtn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>获取Cookie</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="planIdEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="getPlanIdBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>获取PlanId</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CHENRONGHUI</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userPwd">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PA4ZG1NMT04%3D</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_inspect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>抽检</string>
    </property>
    <widget class="QPushButton" name="SignBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>签到</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="resIdEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="longitudeEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="latitudeEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="checkSignBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>签退</string>
     </property>
    </widget>
    <widget class="QPushButton" name="checkSignBtn2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>签退2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timeStartBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动抽检</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="siteNameEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tableIndexEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="auto_task_num_edit">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="console_edit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>291</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="hideBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>隐藏</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
